autoscale/build.gradle

```diff
@@ -28,7 +28,7 @@ dependencies {
     compile group: 'org.springframework.boot',           name: 'spring-boot-starter-actuator',           version: springBootVersion
     compile group: 'org.springframework.boot',           name: 'spring-boot-starter-freemarker',         version: springBootVersion
     compile group: 'org.springframework.boot',           name: 'spring-boot-starter-jersey',             version: springBootVersion
-    compile group: 'org.springframework.security.oauth', name: 'spring-security-oauth2',                 version: '2.0.8.RELEASE'
+    compile group: 'org.springframework.security.oauth', name: 'spring-security-oauth2',                 version: '2.1.0.RELEASE'
```

Comment:
The version should be a variable in gradle.properties, because we use it at two projects